src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -117,10 +170,30 @@ public String getPageStreamingDescriptorName(Method method) {
     return varName(Name.upperCamel(method.getSimpleName(), "PageStreamingDescriptor"));
   }
 
+  public String getPageStreamingDescriptorConstName(Method method) {
+    return inittedConstantName(Name.upperCamel(method.getSimpleName()).join("page_str_desc"));
+  }
+
+  public String getBundlingDescriptorConstName(Method method) {
+    return inittedConstantName(Name.upperCamel(method.getSimpleName()).join("bundling_desc"));
+  }
+
   public void addPageStreamingDescriptorImports(ModelTypeTable typeTable) {
     // do nothing
   }
 
+  public void addBundlingDescriptorImports(ModelTypeTable typeTable) {
```

Comment:
True, but at the same time, it would be somewhat annoying if the new language actually didn't require any imports, and they had to implement an empty method because we forced them to. 

In the end I'm not super concerned with language implementers missing things because we are producing a mechanism to compile and run-check generated surfaces. Even if an implementer is forced to implement a method like this and they add some imports, that doesn't mean they would get the list right; it only means they are *slightly* more likely to get it right.